providers/burst-sms/src/lib/burstSms.provider.ts

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
I missed to ask for a rename of the file to: `burst-sms.provider.ts`. Apologies for that.